addons/website_slides/static/src/xml/activity.xml

```diff
@@ -19,4 +19,20 @@
             </t>
         </t>
     </t>
-</templates>
\ No newline at end of file
+
+    <t t-inherit="mail.Activity" t-inherit-mode="extension">
+        <xpath expr="//*[@name='tools']" position="replace">
+            <t t-if="activity.requestingPartner and activity.thread.model === 'slide.channel'">
+                <div class="o_Activity_tools">
+                    <button class="o_Activity_toolButton o_Activity_GrantAccessButton btn btn-link" t-on-click="_onGrantAccess">
+                        <i class="fa fa-check"/> Grant Access
+                    </button>
+                    <button class="o_Activity_toolButton o_Activity_RefuseAccessButton btn btn-link" t-on-click="_onRefuseAccess">
```

Comment:
`o_Activity_GrantAccessButton` => `o_Activity_grantAccessButton`
`o_Activity_RefuseAccessButton` => `o_Activity_RefuseAccessButton`

_(convention is `o_ComponentName_partName`, so start with small letter for the "part" bit of structural classname)_

Reply:
Done